### PR TITLE
Remove nginx, add .env, add real data, allow all hosts, and more!

### PR DESCRIPTION
**Pull request by cadenforrest**
Suspicion ATM is that nginx is adding to a lot of server-client latency. Disabling until we figure out what's going on, and we'll reinstate it then.

### CODE REVIEW

File: api/backend/backend/urls.py [hunk 1]
```diff
@@ -30,5 +30,6 @@
 urlpatterns = [
     path("api/", include(router.urls)),
     # path('api-auth/', include('rest_framework.urls', namespace='rest_framework'))
+    path("api/divisions/<int:division_id>/league/", views.DivisionLeagueView.as_view(), name="division-league"),
```
Comment (kmorrison) [inline api/backend/backend/urls.py:33]: Request we kill this, I added the `league_info` field to divisions. In general I'd say if we think we need one-off endpoints for one bit of information, let's see if there's a more general fix, very NOT RESTFUL.

File: api/backend/league/views.py [hunk 2]
```diff
@@ -67,3 +68,10 @@ def create(self, request):
             serializer.save()
             return Response(serializer.data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+  
+class DivisionLeagueView(APIView):
+    def get(self,  request, division_id):
+        division = Division.objects.get(id=division_id)
+        league = division.league
+        serializer = LeagueSerializer(league)
+        return Response(serializer.data, status=status.HTTP_200_OK)
```
Comment (kmorrison) [inline api/backend/league/views.py:77]: kill

File: frontend/components/divisions/DivisionsList.tsx [hunk 3]
```diff
@@ -42,7 +43,7 @@ function DivisionCard(division: Division) {
         </Box>
         <Box sx={{ display: 'flex', flexDirection: 'column' }}>
           <CardContent >
-            <Button href={`/divisions/${division.id}/division-info`} LinkComponent={Link}>
+            <Button href={`/divisions/${division.id}/player-list`} LinkComponent={Link}>
```
Comment (kmorrison) [inline frontend/components/divisions/DivisionsList.tsx:46]: mentioned this already, but it's inaccurate to talk about the player list for a division. We could have it default to the current season but seems hacky, let's see if there's a better way to do this, maybe division resource returns yeah, the most recent season and season_division or something via property

Comment (l-ohman) [reply frontend/components/divisions/DivisionsList.tsx:46]: @kmorrison The wireframe that we have for this page is much better suited to _only_ showing data from ongoing/current seasons, but we can adjust if you think we should have the option to display past seasons as well.

Comment (cadenforrest) [reply frontend/components/divisions/DivisionsList.tsx:46]: agreed, but will update in a future PR when we have league / division / season switchers to display historical data, when the season ID gets added to the router query

**Pull request merged**